fish_prompt.fish

```diff
@@ -594,14 +591,39 @@ function __bobthefish_prompt_k8s_context -S -d 'Show current Kubernetes context'
             if [ "$key" = 'current-context:' ]
                 set -l context (string trim -c '"\' ' -- $val)
                 [ -z "$context" ]
-                and return
+                and return 1
 
-                __bobthefish_start_segment $color_k8s
-                echo -ns $context ' '
+                echo $context
                 return
             end
         end <$file
     end
+
+    return 1
+end
+
+function __bobthefish_k8s_namespace -S -d 'Get the current k8s namespace'
+    set -l namespace (kubectl config view --minify --output "jsonpath={..namespace}")
+
+    if [ -z "$namespace" ]
+        echo "default"
```

Comment:
bobthefish tends to omit "default" or "system" whenever possible to keep things clean and only show interesting information. for example, the `master` branch name is omitted when looking at a git repo. does it make sense to leave `default` off here?

Reply:
Yes. Let's omit `default`.